custom-namespaces.yaml

```diff
@@ -118,3 +118,39 @@ contextual_services:
       type: ping_explore
       views:
         base_view: topsites_impression
+mozilla_vpn:
```

Comment:
Hm, we have a `-` `_` mismatch here. This will create a new `mozilla_vpn` directory. It looks like the model is called `mozilla_vpn.model.lkml`, maybe we should deploy this as-is and then copy everything in `mozilla-vpn` over to the new directory once it's deployed.

Reply:
> maybe we should deploy this as-is and then copy everything in mozilla-vpn over to the new directory once it's deployed.

for consistency with other projects, i'm in favor of this approach